rootfs/etc/cont-init.d/01-dumpvdl2

```diff
@@ -20,13 +20,6 @@ read -ra SPLIT_FREQS <<< "${FREQUENCIES}"
 
 # loop through SPLIT_FREQS
 
-# We can only have 8 total frequencies
```

Comment:
https://github.com/szpajder/dumpvdl2#features

It can only decode 8 channels, unless the decoder documentation is wrong. I'd like to leave that check in.

Reply:
Fair enough, we can hold off on merging this until szpajder/dumpvdl2#27 is merged, or split into two PRs

Reply:
@fredclausen the dumpvdl2 PR was merged into unstable

Reply:
I'm happy to merge this in. I don't think upstream has merged your patch in to main yet, but I don't suppose that functionally really matters for us.